packages/flutter/test/semantics/semantics_update_test.dart

```diff
@@ -249,6 +251,13 @@ class SemanticsUpdateBuilderSpy extends ui.SemanticsUpdateBuilder {
       additionalActions: additionalActions,
     );
   }
+
+  @override
+  void updateCustomAction({required int id, String? label, String? hint, int overrideId = -1}) =>
+    _builder.updateCustomAction(id: id, label: label, hint: hint, overrideId: overrideId);
+
+  @override
+  ui.SemanticsUpdate build() => _builder.build();
```

Comment:
This ends up causing some pain if we have to make changes upstream that add a method.

Can we override noSuchMethod or extend Fake on this?

Reply:
Added `extends Fake`